api/src/service/email-service.ts

```diff
@@ -0,0 +1,38 @@
+import nodemailer, { SendMailOptions } from 'nodemailer';
+import {EMAIL_HOST, EMAIL_PORT, EMAIL_USER, EMAIL_PASS} from '../config/keys';
+
+
+export const sendEmail = async (to: string, subject: string, msg: string, icsFileContent?: string) => {
+    try {
+        const transporter = nodemailer.createTransport({
+            host: EMAIL_HOST,
+            port: EMAIL_PORT,
+            secure: true,
+            auth: {
+                user: EMAIL_USER, 
+                pass: EMAIL_PASS 
+            }
+        });
+        
+        const emailOptions = <SendMailOptions> {
+            from: `"Vita" <${EMAIL_USER}>`,
+            to,
+            subject,
+            text: msg
```

Comment:
Maybe change `text: msg` to `html: msg`.
It would be nice if we could render some html in the email too!